src/OrbitGl/TimeGraph.cpp

```diff
@@ -620,7 +622,8 @@ void TimeGraph::RequestUpdatePrimitives() {
 }
 
 // UpdatePrimitives updates all the drawable track timers in the timegraph's batcher
-void TimeGraph::UpdatePrimitives(PickingMode picking_mode) {
+void TimeGraph::UpdatePrimitives(Batcher* /*batcher*/, uint64_t /*min_tick*/, uint64_t /*max_tick*/,
```

Comment:
Hm, ok, this actually shows another issue with this hierarchy, it's very weird that the arguments of `UpdatePrimitives` are ignored for the TimeGraph. But I think it's ok in the current state, and we should consider splitting this part of the functionality from the actual Rendering Container later, WDYT? We should open a bug for that.

Reply:
Agree. Created b/184812692.